astroquery/nasa_exoplanet_archive/nasa_exoplanet_archive.py

```diff
@@ -104,6 +112,9 @@ def query_planet(self, planet_name, table_path=None):
         table_path : str (optional)
             Path to a local table file. Default `None` will trigger a
             download of the table from the internet.
+        all_columns : bool (optional)
+            Return all available columns. The default returns only the
+            columns in the default category at the link above.
```

Comment:
could you add an extra empty line here, the docstring doesn't render correctly atm.